joomla/backend-template
# Show message when module was added to dashboard

Pull Request for Issue # 

### Summary of Changes
When you add a module to the dashboard, nothing happens until you reload the page. Now at least a message shows up. The permissions were not correctly checked and an account with not enough permissions hat problems after the modal opened. That is fixed in this PR too.

### Testing Instructions
Go to the dashboard and click on 'add dashboard module'. After successfully doing that, a message appears. When you have not permissions to create or manage a module, you do not see this button.
